plugins/org.eclipse.sirius.doc/doc/Release_Notes.textile

```diff
@@ -6,7 +6,11 @@ This document contains the release notes for recent major releases of Sirius. Se
 
 h2(#sirius7.5.0). Changes in Sirius  Desktop 7.5.0
 
-Nothing.
+h3. User-Visible Changes
+
+* <span class="label label-info">Modified</span> Edition using the Grid is enhanced. 
+** When creating a node with active "Snap To Grid", the size is adapted to the Grid Spacing so the resizable sides fit on it. This behavior results in having corners of the node matching the grid. Currently, it only applies if the node mapping has size indications to provide a preferred size.
```

Comment:
```suggestion
** When creating a node with "Snap to Grid" enabled, its size is adjusted to the grid spacing so that its resizable sides align with the grid. This ensures that the node’s corners match the grid intersections. Currently, this only applies when the node mapping provides size hints to define a preferred size.
```

Reply:
Ideally, this first part should only have concerned the previous PR #581. Something to keep in mind (and to update) if only the first PR gets merged.